numpy/core/fromnumeric.py

```diff
@@ -3647,13 +3654,13 @@ def std(a, axis=None, dtype=None, out=None, ddof=0, keepdims=np._NoValue, *,
 
 
 def _var_dispatcher(a, axis=None, dtype=None, out=None, ddof=None,
-                    keepdims=None, *, where=None):
+                    keepdims=None, *, where=None, mean=None):
     return (a, where, out)
```

Comment:
Same here, `return (a, where, out, mean)`

Reply:
I'll make the change.